### ISSUE TITLE
HTML

### ISSUE THREAD

**Issue by Axkaban** [DTownz/prj-rev-bwfs-dasmoto#1]
Good use of classes [https://github.com/DTownz/prj-rev-bwfs-dasmoto/blob/master/Arts%26Crafts/index.html#L12](https://github.com/DTownz/prj-rev-bwfs-dasmoto/blob/master/Arts%26Crafts/index.html#L12)

Interesting choice to using anchor tags to separate the paragraph: [https://github.com/DTownz/prj-rev-bwfs-dasmoto/blob/master/Arts%26Crafts/index.html#L15](https://github.com/DTownz/prj-rev-bwfs-dasmoto/blob/master/Arts%26Crafts/index.html#L15)
It is not wrong, it is uncommon, but it makes sense if you were to think that is actually a link to purchase!
for just separating styles of text inside the same `p` element, you can use a commonly use tag: `span` or if you wouldn't want to write the property `font-weight` on that selection, you can use `strong` tags that automatically applies that property.

**Comment by DTownz** [comment]
So I can use span to separate paragraphs? My goal there with he anchor tag was just add a separate class so I can style just that section. How would I go about styling those price sections without the use of anchors

**Comment by Axkaban** [comment]
Yes, that is one of the purposes of the `<span></span>` tag, to make that separation in your `p` element, then you can just use your `span` selector in the css file, or give it a class and target the class.
